fuzzers/013-clb-ncy0/top.py

```diff
@@ -92,7 +101,6 @@ def gen_slices():
     parameter N=-1;
 
     wire [3:0] o;
-    assign dout[0] = o[1];
```

Comment:
you should probably remove both of these assigns if you remove one. Otherwise you introduce aliasing issues. Not a big deal if its working, but its what I did on my local test, and I feel its safer

Reply:
You are right - done